### PR TITLE
Adding custom font and colors assists 🎨

### PR DESCRIPTION
**Pull request by ziadali22**

### CODE REVIEW

File: Dokan/Dokan/Assets.xcassets/Blue Ocean.colorset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (ahmdmhasn) [inline Dokan/Dokan/Assets.xcassets/Blue Ocean.colorset/Contents.json]: Do we really need all of these colors?

File: Dokan/Dokan/Info.plist [hunk 2]
```diff
@@ -2,6 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>DMSans-Regular.ttf</string>
```
Comment (ahmdmhasn) [inline Dokan/Dokan/Info.plist:7]: Did you heard about dynamic font? 😁

**Review by ahmdmhasn — COMMENTED** [review]
🚀

**Review by ahmdmhasn — APPROVED** [review]
🚀🚀🚀

**Review by oradwanomar — APPROVED** [review]

**Comment by ahmdmhasn** [comment]
We may merge this directly into `develop` since it only contains colors and fonts resources. 
@ziadali22

**Pull request merged**